api/commands/query.ts

```diff
@@ -30,11 +34,30 @@ export function run(
 
 export async function evaluate(
   dbadapter: dbadapters.IDbAdapter,
-  query: string,
-  compileConfig?: dataform.ICompileConfig
-): Promise<dataform.IQueryEvaluationResponse> {
-  const compiledQuery = await compile(query, compileConfig);
-  return await dbadapter.evaluate(compiledQuery);
+  queryStringOrTable: string | dataform.ITable,
```

Comment:
i'm not sure this is the right place to do this.

IMO we should be moving away from `query.ts` (which is just a hacky way of injecting arbitrary SQL into the rest of a graph).

I'd expect to instead see us change the `IDbAdapter.evaluate` interface to do something like this. and when evaluating a table/assertion/operation, we'd call that interface (probably exposed somewhere higher level in `api/`), and this function would only really be used for 'selected query' evaluation.

(i would also expect the interface to change to allow all three action types, otherwise the caller has to branch on the action type which is being evaluated).